pkg/types/gcp/validation/platform_test.go

```diff
@@ -1,6 +1,7 @@
 package validation
 
 import (
+	"github.com/openshift/installer/pkg/types"
```

Comment:
nit: can you move this to the final import block?

Imports should be

<stdlib>

<external>

<internal>